### PR TITLE
CA-370 New SERP: transfer the reusable components to IM-components repo

### PR DESCRIPTION
**Pull request by Maxusha**
CA-370 New SERP: transfer the reusable components to IM-components repo

Besides components described in task few extra had to be transferred due to dependencies. Some components (like `Label`) has parameters changed. So everything has to be reviewed carefully.

List of touched components:
- button
- car tile
- checkbox
- dropdown
- footer (added envkv)
- icon
- input
- label
- rangeDropdown
- styledDropdown (added)
- typography

### CODE REVIEW

**Comment by bezvoshchuk** [comment]
Looks good to me, but there are conflicts tryin' to rebase to latest master. Please resolve conflicts.

**Comment by raqystyle** [comment]
5 cents from me:
- Please, run the `bump:version` command. It will generate all the necessary changelogs and will bump the versions. It will also delete the temporary files. Ideally, this command should be run on CI. If we had this implemented, you would have had a well-formed PR. But, for the time being, we need to run that command ourselves.

File: .changeset/61afca20/changes.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (raqystyle) [inline .changeset/61afca20/changes.json]: Please, run also `bump:version`, I commented on this in the `Conversation` page.

Comment (Maxusha) [reply .changeset/61afca20/changes.json]: Done in 7a72b48

**Pull request merged**